bindings/python/examples/how_tos/outputs/output_unlock_conditions.py

```diff
@@ -0,0 +1,86 @@
+from iota_sdk import *
+from dotenv import load_dotenv
+import json
+
+load_dotenv()
+
+client = Client()
+
+hex_address = Utils.bech32_to_hex(
+    'rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy')
+
+alias_hex_address = Utils.bech32_to_hex(
+    'rms1pr59qm43mjtvhcajfmupqf23x29llam88yecn6pyul80rx099krmv2fnnux')
+
+address_unlock_condition = AddressUnlockCondition(
+    Ed25519Address(hex_address)
+)
+
+token_scheme = TokenScheme(0, 50, 100)
```

Comment:
A bit confusing that the order is different in Python, but we can change that in another PR

Reply:
Yeah true that. Thought about the same. Let me quickly create an issue for that so it doesn't get forgotten

Reply:
https://github.com/iotaledger/iota-sdk/issues/474